packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -258,6 +260,7 @@ const BaseLink = ({
       motionDuration={motionDuration}
       motionEasing={motionEasing}
       setCurrentInteraction={setCurrentInteraction}
+      // @ts-ignore
```

Comment:
Why do we need this?

Reply:
It's because of this: https://github.com/razorpay/blade/pull/675/files/21e5af143380e3c1117e77b1114c873633dfeaf3#diff-8df6f3ac8deb957fe315382616c9e40354240ff966d0f5f089583dade523a644R196-R198

(This was added earlier so that we can extend the BaseLink with `styled(BaseLink)` but still don't have to expose the typescript API to public, [reference](https://github.com/razorpay/blade/pull/587#discussion_r938456767))